### PR TITLE
[13.0][MIG] barcodes_generator_location: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27167

### CODE REVIEW

File: barcodes_generator_location/__manifest__.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+    "version": "13.0.1.0.0",
     "category": "Tools",
-    "author": "LasLabs, " "Odoo Community Association (OCA)",
-    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-barcode",
     "license": "AGPL-3",
-    "depends": ["barcodes_generator_abstract", "stock",],
-    "data": ["views/stock_location.xml",],
+    "depends": ["barcodes_generator_abstract", "stock", "stock_barcodes"],
```
Comment (pedrobaeza) [inline barcodes_generator_location/__manifest__.py]: This dependency shouldn't be used. It's too much for this.

Comment (ernestotejeda) [reply barcodes_generator_location/__manifest__.py]: @pedrobaeza  take a look at the changes I made regarding this, please.

Comment (chienandalu) [reply barcodes_generator_location/__manifest__.py]: Would it be more appropiate to pull an extra `stock_location_barcode` module which both this and `stock_barcodes` depends on to keep this stuff simple?

Comment (pedrobaeza) [reply barcodes_generator_location/__manifest__.py]: I don't think we should add such module complexity, and more not having control over enterprise module `stock_barcode`, so better to inject it via code if missing.

**Comment by victoralmau** [comment]
It's ready to review because `stock_barcodes` dependency have removed, right?

**Comment by pedrobaeza** [comment]
Please finish it, whatever it takes.

**Review by victoralmau — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu is it OK for you?

**Comment by chienandalu** [comment]
I'll do the final functional review

**Review by chienandalu — APPROVED** [review]
Works as expected :+1:

**Pull request merged**